types/datum_test.go

```diff
@@ -173,10 +173,6 @@ func (ts *testTypeConvertSuite) TestToInt64(c *C) {
 	v, err := Convert(3.1415926, ft)
 	c.Assert(err, IsNil)
 	testDatumToInt64(c, v, int64(3))
-
-	binLit, err := ParseHexStr("0x9999999999999999999999999999999999999999999")
-	c.Assert(err, IsNil)
-	testDatumToInt64(c, binLit, -1)
```

Comment:
Why delete this case? We should correct it if the expected result was wrong.

Reply:
I've moved this test to other place, and cast(0x99999999999999999999999999) to -1 will not happen in this fuction.